rentals.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :person, :book, :date
+
+  def initialize(person, book, date)
+    @person = person
+    @book = book
+    @date = date
+    @person.add_rental(self)
+    @book.add_rental(self)
+  end
```

Comment:
- [ ] It would be nice if you make your initialize something like below to complete your many-to-many relationship :blush: because even after adding rentals in attr_accessors it is still not enough. You still need to make your initialize section something like the below :arrow_down:

```ruby
  def initialize(person, book, date)
    @person = person
    @book = book
    @date = date
    @person.rentals.push(self)
    @book.rentals.push(self)
  end
```